modules/assim.sequential/inst/WillowCreek/SDA_Workflow_LAI.R

```diff
@@ -0,0 +1,345 @@
+# ----------------------------------------------------------------------
```

Comment:
This seems to be an entirely new file that you're adding as part of the PR. Would prefer if you hold this back for a separate PR

Reply:
Yes, sorry that was a mistake it must've got added when I was switching branches because I still had the script open. I will remove it.